executor/common_fuchsia.h

```diff
@@ -176,13 +176,22 @@ long syz_mmap(size_t addr, size_t size)
 		fail("zx_object_get_info(ZX_INFO_VMAR) failed: %d", status);
 	zx_handle_t vmo;
 	status = zx_vmo_create(size, 0, &vmo);
-	if (status != ZX_OK)
+	if (status != ZX_OK) {
+		fprintf(stderr, "zx_vmo_create failed with: %d", status);
 		return status;
+	}
+	status = zx_vmo_replace_as_executable(vmo, ZX_HANDLE_INVALID, &vmo);
+	if (status != ZX_OK) {
+		fprintf(stderr, "zx_vmo_replace_as_executable failed with: %d", status);
```

Comment:
Same here.
Also please no {} around blocks with 1 statement. FWIW it's the style this code base uses throughout.
Also, shouldn't this syscall fail in this case and return non-zero status? If this will start failing when called from os_init nobody will ever notice this and see this error.